### PR TITLE
update route functions

### LINKED ISSUE #21: Update Route
add queries functions to routes

### PR DESCRIPTION
**Pull request by yasmeenattallah**
update comment and post route

### CODE REVIEW

File: src/controllers/comments.js [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
 
 const createComment = (req, res, next) => {
   const { commentContent } = req.body;
+  postComments(commentContent)
+    .then(({ rows }) => {
+      res.status(200).json({ rows });
+    })
+    .catch(next(errCatcher('EMPTY input...', 400)));
+
   if (!commentContent) {
```
Comment (mAbadsa) [inline src/controllers/comments.js]: Move up postComments

File: src/controllers/posts.js [hunk 2]
```diff
@@ -1,26 +1,37 @@
...
 };
 
 const createPosts = (req, res, next) => {
   const { postContent } = req.body;
+  postPosts(postContent)
+    .then(
+      ({ rows }) => res.status(200).json({ rows }),
+    )
+    .catch(next(errCatcher('something went wrong...', 400)));
   if (!postContent) {
```
Comment (mAbadsa) [inline src/controllers/posts.js]: Move up

File: src/controllers/posts.js [hunk 3]
```diff
@@ -1,26 +1,37 @@
...
 };
 
 const updatePost = (req, res, next) => {
   const { postContent } = req.body;
+  const { postId } = req.params;
+  updateData(postId, postContent)
+    .then(({ rows }) => res.status(200).json({ rows }))
+    .catch(next(errCatcher('something went wrong...', 400)));
+
   if (!postContent) {
```
Comment (mAbadsa) [inline src/controllers/posts.js:27]: Move up

**Review by AlaaShurrab — CHANGES_REQUESTED** [review]
you should make the error catching before running the queries so that it doesn't make an issue in the database

File: src/controllers/comments.js [hunk 4]
```diff
@@ -1,16 +1,26 @@
...
+    })
+    .catch(errCatcher('NOT FOUND  ...', 400));
 };
 
 const createComment = (req, res, next) => {
   const { commentContent } = req.body;
   if (!commentContent) {
-    // throw new Error('Empty input...');
     next(errCatcher('EMPTY input...', 400));
+  } else {
```
Comment (mAbadsa) [inline src/controllers/comments.js]: does not need else here

File: src/controllers/posts.js [hunk 5]
```diff
@@ -1,26 +1,37 @@
...
-  return res.status(200).json(postContent);
 };
 
 const updatePost = (req, res, next) => {
   const { postContent } = req.body;
+  const { postId } = req.params;
   if (!postContent) {
-    // throw new Error('Empty input...');
     next(errCatcher('EMPTY input...', 400));
+  } else {
```
Comment (mAbadsa) [inline src/controllers/posts.js]: does not need else here

**Review by mAbadsa — CHANGES_REQUESTED** [review]

**Review by mAbadsa — APPROVED** [review]

**Review by AlaaShurrab — APPROVED** [review]

**Pull request merged**